README_new.md

```diff
@@ -0,0 +1,132 @@
+
+We also want to hire people who are **reasonable**, **professional**, and **fun** to work with. After all, they are people we will spend a great deal of time interacting with on a daily basis.
+
+These are the main areas we in the iOS team will focus our assessment of candidates on:
+
+- Technical expertise
+ - Languages (Swift, Objective-C) and platforms (iOS, watchOS, tvOS, macOS)
+ - Architectural design patterns (MVC, MVVM, Viper, etc)
+ - Knowledge of iOS APIs
+ - Knowledge of the development cycle
+ - Knowledge of reactive programming
+ - Interview demo project
+- Interest in sharing one's knowledge
+- Interest in learning new knowledge
+- Interest in keeping oneself up to date
+- Ability to think and work independently
+- Ability to work in a large team
+- Professional and personal maturity
+- Overall attitude
+- Likelihood of fitting in well with the company's culture
+- Is the candidate someone we would like to work with on a daily basis?
+
+### The iOS recruitment process
+
+Our recruitment process is different for candidates who are interested in working remotely compared to the process for candidates who are interested in working on-site at our office in London:
+ - on-site: Initial call -> Demo -> 3-hour face-to-face technical interview
+ - remote: Initial call -> Demo -> one or more Skype interviews -> 1 working day with us (at our expense)
+
+As indicated above, we expect candidates to submit an "interview demo project" but, in order to be flexible, we provide 3 different ways in which a candidate can submit a demo of their work. **You only need to do _one_ of these**.
+
+1. [Do the babylon demo project](#1-babylon-demo-project)
+2. [Send us a project you've already done](#2-already-written-project)
+3. [Show us some open source work you did](#3-open-source-work)
+
+Keep in mind that the demo project should be an example of one's **best** work. We want to be wowed by the candidate's techincal skills and creativity.
+
+If we think the demo project is interesting enough, we'll proceed to a face-to-face step, either on Skype (for remote candidates) or in person (for on-site candidates).
+
+It typically starts with a discussion around the candidate's project. We normally ask the candidate to implement a new feature (even if it's using pseudo-code), or to refactor and unit-test a particular component, and in general we aim to have an informal discussion about the candidate's approach.
+
+We may also, or instead of, focus on more technical aspects and ask a number of highly technical questions. As indicated earlier, we have open-sourced these questions as well.
+
+This may look like a radical departure from the typical recruitment process but we think that it gives candidates a fair chance to prepare themselves and it also makes assessment uniform across many candidates.
+
```

Comment:
I don't see any value in comparing our recruitment process with recruitment processes by other companies. I would reformulate this to say what we hope to achieve with our recruitment process. Maybe something like:

We believe that a fully open recruitment process gives all candidates a fair chance to prepare, and make our assessment uniform across all candidates.

Reply:
Fair point. I like your suggested text!

Reply:
Fixed in https://github.com/Babylonpartners/iOS-Interview-Demo/pull/4/commits/5604f8b31b3d65a2b940a3323138f57faf9598f3.